api/v2/views/base.py

```diff
@@ -199,8 +200,9 @@ def perform_update(self, serializer):
             raise exceptions.ParseError(detail=message)
         except Exception as e:
             message = {
-                "An error was encoutered when updating the request."
+                "An error was encoutered when updating the request: %s" % e.message
             }
+            logger.exception(e)
```

Comment:
the Exception (e) should include ExceptionType && message, as well, because its a `logger.exception` the traceback will be included by default.

In the past this is generally enough information to catch the error